src/elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip.scss

```diff
@@ -0,0 +1,3 @@
+.bdl-AdditionalTabFtuxTooltip.tooltip {
+    max-width: 250px;
```

Comment:
This value seems kind of magic number-y. Is it sized relative to something else?

Reply:
The default width was too wide to fit on the right hand side of the sidebar, and we don't have official designs to work from, so I just picked a number that looked good and fit in the sidebar. Is there a better place to get this number from?

Reply:
There's `$sidebarContentWidth` available as a variable somewhere, but it's larger than what you have here. Maybe just leave a comment explaining the reason for the value?